### DISCUSSION TITLE
remark>=14 for gatsby-plugin-mdx

### DISCUSSION THREAD

**Discussion by yklcs** [gatsbyjs/gatsby#30379] [Ideas / Feature Requests] [answered]
[`remark>=14`](https://github.com/remarkjs/remark/releases/tag/13.0.0) brings a new engine (`micromark`).

While this doesn't change `mdx` rendering functionality, latest `remark` plugins require `remark>=14` to use (e.g. [`remark-math`](https://unifiedjs.com/explore/package/remark-math/#important)).

Currently, `gatsby-plugin-mdx` is using `remark^10`. Are there any plans to bump the `remark` version? I would put in a PR, but there are too many related plugins.

**Comment by LekoArts** [comment, marked as answer]
You can follow https://github.com/gatsbyjs/gatsby/pull/29678 for progress

**Comment by kimbaudi** [comment]
@RocketLL - I'm facing the exact situation where I would like to upgrade `remark-math` from v3 to v4 for `gatsby-plugin-mdx`. However, `gatsby-plugin-mdx` cannot upgrade to remark >= 13 until `mdx-js/mdx` package is updated to v2. It is currently in beta. see https://github.com/mdx-js/mdx/issues/1440#issuecomment-779388382.

**Reply by yklcs** [reply]
Yup, I also saw that it is being blocked by MDXv2 👍
